driver-core/src/test/java/com/datastax/driver/core/AtomicMonotonicTimestampGeneratorTest.java

```diff
@@ -29,10 +30,58 @@
 import java.util.concurrent.Executors;
 import java.util.concurrent.TimeUnit;
 
+import static org.assertj.core.api.Assertions.assertThat;
 import static org.testng.Assert.assertEquals;
 import static org.testng.Assert.fail;
 
-public class AtomicMonotonicTimestampGeneratorTest {
+public class AtomicMonotonicTimestampGeneratorTest extends CCMTestsSupport {
```

Comment:
Could we keep it unit test only? It pains me that we have to spin up a Cassandra cluster to test such a simple component.